packages/app-content-pages/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.2'
+
+services:
+  dev:
+    build:
+      context: ../../
+    entrypoint:
+      - "yarn"
+      - "workspace"
+      - "@zooniverse/fe-content-pages"
+    command: ["dev"]
+    ports:
+      - "3000:3000"
+      - "9001:9001"
+    environment:
+      - HOST=0.0.0.0
+      - AWS_REGION=${AWS_REGION}
```

Comment:
afaik none of these AWS env vars are required. you can see the required vars in `env-default` - and these don't need to be set here anyway, since [docker compose supports `.env` files](https://docs.docker.com/compose/environment-variables/#the-env-file)

Reply:
Ok. This is just copied from the docker-compose file we use to run the project app.

Reply:
I think HOST does need to be set, though.

Reply:
webpack dev server doesn't work without HOST but express is fine. 🤷‍♂ computers, eh?